lib/project_types/script/commands/connect.rb

```diff
@@ -5,6 +5,9 @@ class Connect < ShopifyCLI::Command::SubCommand
       prerequisite_task :ensure_authenticated
       prerequisite_task ensure_project_type: :script
 
+      recommend_default_node_range
```

Comment:
@jacobsteves what Node versions do we expect people to have to develop scripts? Do we have tests to ensure we support those?

Reply:
Thanks Jacob. I adjusted the check to use as the minimum version the value of the constant that currently has the 14.5.0 value.